### ISSUE TITLE
[20220515] Weekly AI ArXiv 만담 - 51회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#51]
### News
- Conference
  - NeurIPS deadline: Abs 5.16 (우리시간 5.17 05:00AM), Full 5.19 (우리시간 5.20, 05:00AM), Supp 5.26 (우리시간 5.27, 05:00AM)
  - ICML decision: 5.14 AoE (우리시간 5.15, 21:00 예정 but 오후에 공개): 모두들 축하드립니다.
- Google I/O
  - https://io.google/2022/intl/ko/
  - Multisearch, Google Glass, Pixel Watch, New Google Maps, 웨이크업 없는 AI 스피커 등
  - 요약: https://www.xda-developers.com/google-io-2022-recap-major-announcements/
- [IT 기술기업 스타트업 투자 가을이 오는가?](https://n.news.naver.com/mnews/article/031/0000672507?sid=105)
- Meta OPT 공개 (정말 모든걸 공개)
- [HyperscaleFAccT CRAFT @ ACM FAccT 2022](https://naver-career.gitbook.io/en/teams/clova-cic/events/hyperscalefacct-facct-2022)
  - 6.21, 서울 코엑스. 
  - 고학수 교수님, 조경현 교수님, M. Mitchell (Ethical AI), 차미영 교수님, 박상철 교수님, Deep Ganguli (Anthropic), 배순민 소장님 등
- 국내 학회
  - [2022 한국IT서비스학회 춘계학술대회](https://www.itservice.or.kr/2022/conference_1.html): 5.18, 대한상공회의소
### ArXiv
- [Unifying Language Learning Paradigms](https://arxiv.org/abs/2205.05131)
  - LM을 위한 통합 학습기법: Dec only도 Enc-Dec 도 모두 가능 (from Google Research, PaLM팀)
  - Pretrainig task: Mixture of Denoisers (S, R, X)
  - Denosing mode 변환을 위해 paradigm token 활용
  - 그래서 Finetuning 에 서도  in-context learning에서도 더 좋은 성능
  - 아쉬운 점: 얘는 비교를 하려면 Chinchilla 비슷한 크기로 맞춰서 비교했어야..
  - 적당한 크기 (수십B 이하) 에서 파라미터를 굉장히 잘활용해서 좋은 LM을 만드는 기법들 (좋은 소식)
![image](https://user-images.githubusercontent.com/11782739/168471149-ede1d40a-5b56-4d01-bd93-06ea86965d7c.png)
![image](https://user-images.githubusercontent.com/11782739/168471165-8421e38d-d23e-472d-ae02-cbeed1917c7e.png)

 
- [A Generalist Agent](https://www.deepmind.com/publications/a-generalist-agent)
  - Gato: 604개의 RL게임, Vision 기반 Robot control, Text, image-text 캡셔닝 등 604개의 task를 동일한 웨이트 하나의 1.2B Transformer 로 다루는 제목 그대로 generalist agent (from Deepmind)
  - 모든 입력은 token으로 변환 (텍스트, 이미지, discrete value, continuous value), Voc size는 대략 35000
  - output은 target token prediction 그래서 image 와 agent observation token prediction task는 없음
  - 이미지는 ResNet ecoder로 임베딩, 나머지는 모두 lookup table 스타일 임베딩
  - task 구분없이 동일한 형태의 token 시퀀스로 다 집어넣고 학습
  - 동일 도메인내 다양한 태스크 구분을 위해서는 prompt 를 활용
  - 16 x 16 TPU v3 512batch 1M step 4일간 학습 (의외로 computation 많이 안들어가는 듯)
  - 나머지 각 도메인 태스크 결과는 논문에서 확인 가능하십니당.
![image](https://user-images.githubusercontent.com/11782739/168467581-e5adc89e-d1a0-4652-8599-9af4d1bc2e9c.png)
![image](https://user-images.githubusercontent.com/11782739/168467593-11f4d9a4-c596-4da4-aaf5-7259c7579239.png)
![image](https://user-images.githubusercontent.com/11782739/168467603-32ce2b7e-89f1-4ce9-afcb-02fcdbcbefab.png)

- 눈여겨볼 논문들
  - [One Model, Multiple Modalities: A Sparsely Activated Approach for Text, Sound, Image, Video and Code](https://arxiv.org/abs/2205.06126)
    - Tecent 에서 나온 하나의 모델로 다양한 모달리티 데이터를 기법
    - MoE이긴 한데 모달리티 별로 분할 activation 시키는 구조로 보임.

**Comment by ghlee0304** [comment]
### Arxiv
- 눈여겨볼 논문들
    - [Muskits: an End-to-End Music Processing Toolkit for Singing Voice Synthesis](https://arxiv.org/abs/2205.04029)
    - [Vocalsound: A Dataset for Improving Human Vocal Sounds Recognition](https://arxiv.org/abs/2205.03433) 
    - [Silence is Sweeter Than Speech: Self-Supervised Model Using Silence to Store Speaker Information](https://arxiv.org/abs/2205.03759)

**Comment by kimyoungdo0122** [comment]
### News
- Hugging Face가 1억 달러 시리즈 C 투자를 유치했습니다
    - [Hugging Face reaches $2 billion valuation to build the GitHub of machine learning](https://techcrunch.com/2022/05/09/hugging-face-reaches-2-billion-valuation-to-build-the-github-of-machine-learning/)
    - 시리즈 A에서는 1천 500만 $, B에서 4천만 $, C에서 1억 $ 투자를 받았고 현재는 기업가치가 20억 달러에 이른다는 내용
    - 허깅페이스의 BM은 SaaS? 
    - 수많은 pre-trained model을 보유한 허깅페이스가 AI community에 이미 엄청난 영향력을 갖고 있는데,
    -  infrastructure solution이 곁들여진다면 Large-scale auto ml 분야와 같은 enterprise에도 큰 영향력을 가질 것 같습니다

**Comment by veritas9872** [comment]
**Better plain ViT baselines for ImageNet-1k**

Arxiv: https://arxiv.org/pdf/2205.01580.pdf
GitHub: https://github.com/google-research/big_vision/tree/main/big_vision

ViT 학습이 기존 통념보다 잘 된다고 합니다. 기존 CNN에 사용하던 테크닉 없이 단순한 학습 방법을 적용했을 때 오히려 ViT 모델이 성능이 더 좋다는 구글의 연구 결과입니다. ViT 모델이 문제없이 ImageNet top-1 80%를 300 epoch 만에 찍는 것을 보여줍니다. 결론은 너무 어렵게 생각하지 말자는 것...

![Screenshot (52)](https://user-images.githubusercontent.com/33523965/168473005-062d92e8-5c73-421f-8009-0c0c2c32fbe2.png)

![Screenshot (53)](https://user-images.githubusercontent.com/33523965/168472997-490c097e-163f-4881-92f7-a727392e93f4.png)


**CoCa: Contrastive Captioners are Image-Text Foundation Models**

![Screenshot (54)](https://user-images.githubusercontent.com/33523965/168473456-72c2b295-2b06-402c-8bd8-6011d2162443.png)

ImageNet 새로운 SOTA가 등장했습니다. 구글에서 Image to Text 모델을 학습한 결과 모든 라벨을 text 형태로 촐력하게 하여 zero-shot accuracy 86.3%와 fine-tuning 결과 91.0%라는 SOTA를 세웠습니다. ~결론은 역시 돈 많은 자가 승리한다는 것...~

![Screenshot (56)](https://user-images.githubusercontent.com/33523965/168473539-4633c4ec-5ee2-438c-aecc-e3e1455590e9.png)

이번 주는 너무 새로 나온 내용이 많아서 Yannic Kilcher의 YouTube 공유해드립니다. 

ML News Part 1: https://www.youtube.com/watch?v=pwSnC8jlh50
ML News Part 2: https://www.youtube.com/watch?v=smUHQndcmOY

**Comment by veritas9872** [comment]
Tensorboard / W&B 대신 사용할 수 있는 AimStack 공유해드립니다. https://aimstack.io/
